src/SnackbarContainer.tsx

```diff
@@ -4,37 +4,71 @@ import { makeStyles } from '@material-ui/core/styles';
 import { SNACKBAR_INDENTS } from './utils/constants';
 import { SnackbarProviderProps } from '.';
 
-const useStyle = makeStyles(theme => ({
+const collapse = {
+    container: '& > .MuiCollapse-container',
```

Comment:
This CSS class is deprecated. See https://github.com/mui-org/material-ui/pull/24084
And I guess it is removed in v5